test/sequential/test-fs-watch-system-limit.js

```diff
@@ -36,14 +36,27 @@ function spawnProcesses() {
 
 spawnProcesses();
 
+setTimeout(() => {
+  if (!finished) {
+    const timeoutStream = new stream.PassThrough();
+    timeoutStream.write('Error: timeout');
+    timeoutStream.end();
+    timeoutStream.pipe(gatherStderr);
```

Comment:
Why not call `gatherStderr.write()` directly?

Reply:
good call!